pubtools/pulplib/_impl/model/unit/modulemd_defaults.py

```diff
@@ -0,0 +1,28 @@
+from .base import Unit, unit_type
+
+from ..attr import pulp_attrib
+from ... import compat_attr as attr
+
+
+@unit_type("modulemd_defaults")
+@attr.s(kw_only=True, frozen=True)
+class ModulemdDefaultsUnit(Unit):
+    """A :class:`~pubtools.pulplib.Unit` representing a modulemd_defaults document.
+
+    .. versionadded:: 2.4.0
+    """
+
+    name = pulp_attrib(type=str, pulp_field="name")
+    """The name of this modulemd defaults unit"""
+
+    stream = pulp_attrib(type=str, pulp_field="stream")
+    """The stream of this modulemd defaults unit"""
+
+    repository_id = pulp_attrib(type=str, pulp_field="repo_id")
```

Comment:
I think it's the opposite, actually -- this library seems to be pushing "repository_id", whereas I've only seen "repo_id" when dealing with Pulp responses.

Reply:
hmm, if I do `grep repository_id . -r` under pubtools/_impl/, it shows me only one usage of repository_id, which is used as a variable in client.py. Why do you think it's pushing repository_id instead of repo_id?

Reply:
Huh. Okay, yeah. Maintenance and distributor models use "repo_id" while get_repository uses "repository_id". There are also some private methods in client.py using "repo_id", but that's less of a concern.

I really don't know which should be preferred here. I seem to remember @rohanpm mentioning somewhere that he'd like to not use shortened words in these open-source projects, as it could confuse external contributors.

Reply:
Yup, you're right. In Rohan's knowledge share about API design, he suggested not using abbreviated terms, such as repository_id vs repo_id. But it's an exception here, since repo_id has been used before and consistency should be considered first to avoid confusion.